### ISSUE TITLE
[Custom Applications] Missing Applications from the List and from the Search

### ISSUE THREAD

**Issue by simonemainardi** [ntop/ntopng#2535]
Added duplicate application is missing from the list of applications and cannot be searched 

![image](https://user-images.githubusercontent.com/3397663/56196544-51010580-6037-11e9-9bd9-c1ea5ab06441.png)

Even though that application has been written to the custom protocols file

```
(base) 000:ntopng simone$ cat /tmp/protos.txt
# ZATTOO
tcp:3000@ZATTOO
```

**Comment by simonemainardi** [comment]
also non-duplicated applications are missing from the list and from the search

![image](https://user-images.githubusercontent.com/3397663/56196856-f4521a80-6037-11e9-8272-8cfb2b658bca.png)

![image](https://user-images.githubusercontent.com/3397663/56196873-fddb8280-6037-11e9-8eda-c611675ff0eb.png)

![image](https://user-images.githubusercontent.com/3397663/56196880-03d16380-6038-11e9-894a-98cc4cc9d481.png)

**Comment by simonemainardi** [comment]
non-duplicate applications only appear after a restart of ntopng

![image](https://user-images.githubusercontent.com/3397663/56197010-39764c80-6038-11e9-87f4-806cd8c550bb.png)


duplicate applications never show up

**Comment by emanuele-f** [comment]
Duplicate applications are now blocked as a fix of #2534 so this can be closed
